contrib/devtools/update-translations.py

```diff
@@ -159,6 +161,11 @@ def postprocess_translations(reduce_diff_hacks=False):
                 for translation in translations:
                     if translation is None:
                         continue
+
+                    # Check if btc address is included in translation after removing whitespaces
+                    if re.search(ADDRESS_RE, translation.replace(' ', '')):
+                        translation_node.set('type', 'malicious')
+                        break
                     errors = []
                     valid = check_format_specifiers(source, translation, errors, numerus)
```

Comment:
Independent of this PR, but this bit could be simplified a fair bit:
* Return error from `check_format_specifiers`, which generates at most 1 error
* Combine loop and if not valid below